improver/cli/apply_bias_correction.py

```diff
@@ -59,10 +58,9 @@ def process(
     post-bias correction.
 
     Args:
-        forecast_cube (iris.cube.Cube):
-            Cube containing the forecast to apply bias correction to.
-        bias_cubes (iris.cube.Cube or list of iris.cube.Cube):
-            A cube or list of cubes containing forecast bias data over the a specified
+        cubes (iris.cube.Cube or list of iris.cube.Cube):
+            Cube or list of cubes containing the forecast to apply bias correction to, as
+            well as the cubes containing forecast bias data over the a specified
```

Comment:
It would be good to specify here that order of the cubes in the list doesn't matter

Reply:
OK. I've reformatted the comment here to be aligned with the format in `apply_emos_coefficients` CLI. It doesn't specify that any ordering is fine, but it is presented in a way that makes this implicit.

I've also added a warning case in line with that CLI, along with associated tests (as per above). This requires the use of updated acceptance tests data which is in the following PR: https://github.com/metoppv/improver_test_data/pull/47.